test/test_seqr_loader_dry.py

```diff
@@ -161,11 +155,29 @@ def _mock_cohort():
     return cohort
 
 
-def test_seqr_loader_dry(mocker: MockFixture):
+def selective_mock_open(*args, **kwargs):
+    if str(args[0]).endswith('.toml'):
+        # Don't mock calls to load a config file
+        if isinstance(args[0], Path):
+            if len(args) > 1:
+                return open(str(args[0]), *args[1:], **kwargs)
+            return open(str(args[0]), **kwargs)
+        return open(*args, **kwargs)
+    else:
+        return mock_open(read_data="<stub>")(*args, **kwargs)
```

Comment:
Genius. I was making tracks with the `tmp_path`/set_config_paths but this mocked open killed me. Good solution.

Reply:
Yeah, this one had me scratching my head for a while! Got the idea from Stack, bless 'em.